libs/config_registry/include/hpx/config_registry.hpp

```diff
@@ -10,17 +10,33 @@
 #include <string>
 #include <vector>
 
+#if (defined(_WIN32) || defined(__WIN32__) || defined(WIN32))
```

Comment:
Should we use `HPX_WINDOWS` here (after `#include <hpx/config/compiler_specific.hpp>`)?

Reply:
This whole change wasn't meant to be in this PR at all, so I'll remove it. It will be needed later though. The idea is that this module really is at the bottom of the hierarchy (`config` also depends on this) and therefore it has to repeat some functionality, but only the minimum required. It's the same reason why I added the minimal test macro for `config_registry` (https://github.com/STEllAR-GROUP/hpx/blob/62c825ff76ffc3a21588ca2c36730c9c9753aa5a/libs/config_registry/tests/unit/config_registry.cpp#L14-L40).